api/types/journal.go

```diff
@@ -2,3 +2,7 @@ package apitypes
 
 // JournalIDHeader is the name of the HTTP Header used to provide a journal ID to assocate multiple actions.
 const JournalIDHeader = "JournalID"
```

Comment:
Should we impose a regex on valid journal IDs? Asking b/c then we can have a single `JournalHeader` constructed as `<journal_id><sep><journal_description>`.

Reply:
We could, but I don't feel there's a strong reason to.